doc/source/guide/style.rst

```diff
@@ -45,18 +46,19 @@ We now see that the displayed visualizations adopt these new imported settings.
   :align: center 
 
 Similarly, we can change the plot configurations for Matplotlib charts as well.
-The plot_config attribute for Matplotlib charts takes in both the figure and axis as parameters.
+The plotting_style attribute for Matplotlib charts takes in both the figure and axis as parameters.
+
 .. code-block:: python
 	
-	lux.config.plotting_backend = "matplotlib" # or 'matplotlib_code'
+    lux.config.plotting_backend = "matplotlib" # or 'matplotlib_code'
 
     def add_title(fig, ax):
```

Comment:
Can we show the same example as above of changing color and adding title. It might be good to demonstrate how we can change settings for `fig` and `ax` parameter and when we should use each. Also would we be able to set something like `plt.rcParam`? A few examples outlining the possible style changes here with links to matplotlib docs would be helpful here.